proposals/improved-interpolated-strings.md

```diff
@@ -508,19 +499,24 @@ is specified after the interpolated string literal, an error is produced.
 Because `$"{await A()}"` is a valid expression today, we need to rationalize how interpolation holes with await. We could solve this with a few rules:
 
 1. If an interpolated string used as a `string`, `IFormattable`, or `FormattableString` has an `await` in an interpolation hole, fall back to old-style formatter.
-2. If an interpolated string is subject to an _implicit\_string\_builder\_conversion_ and _applicable\_interpolated\_string\_builder\_type_ is a `ref struct`, `await` is not allowed to be used
+2. If an interpolated string is subject to an _implicit\_string\_handler\_conversion_ and _applicable\_interpolated\_string\_handler\_type_ is a `ref struct`, `await` is not allowed to be used
 in the format holes.
 
 Fundamentally, this desugaring could use a ref struct in an async method as long as we guarantee that the `ref struct` will not need to be saved to the heap, which should be possible if we forbid
 `await`s in the interpolation holes.
 
-Alternatively, we could simply make all builder types non-ref structs, including the framework builder for interpolated strings. This would, however, preclude us from someday recognizing a `Span`
+Alternatively, we could simply make all handler types non-ref structs, including the framework handler for interpolated strings. This would, however, preclude us from someday recognizing a `Span`
 version that does not need to allocate any scratch space at all.
 
-### Builders as ref parameters
+_Answer_:
+
+We will treat interpolated string handlers the same as any other type: this means ref struct usage in holes is disallowed. The spec around lowering of string literals used as strings
```

Comment:
> this means ref struct usage in holes is disallowed

I don't understand this.  Can you elaborate?

Reply:
Let me reword and elaborate, I think I didn't do a good job being clear.